Modules/Loadable/Markups/MRML/vtkMRMLMarkupsNode.cxx

```diff
@@ -222,7 +222,7 @@ void vtkMRMLMarkupsNode::ProcessMRMLEvents(vtkObject *caller,
                                            unsigned long event,
                                            void *callData)
 {
-  if (caller != nullptr && event == vtkMRMLTransformableNode::TransformModifiedEvent)
+  if (this->Scene && !this->Scene->IsImporting() && caller != nullptr && event == vtkMRMLTransformableNode::TransformModifiedEvent)
```

Comment:
Should we also block the similar update in vtkMRMLMarkupsCurveNode::ProcessMRMLEvents?